src/core/components/select/Select.tsx

```diff
@@ -0,0 +1,122 @@
+import { SelectHTMLAttributes } from 'react';
+import { InlineError, InlineSuccess } from '@guardian/src-user-feedback';
+import { Label } from '@guardian/src-label';
+import {
+	select,
+	selectWrapper,
+	errorChevron,
+	successChevron,
+	errorInput,
+	successInput,
+} from './styles';
+import { Props } from '@guardian/src-helpers';
+import { SvgChevronDownSingle } from '@guardian/src-icons';
+
+import {
+	visuallyHidden as _visuallyHidden,
+	descriptionId,
+	generateSourceId,
+} from '@guardian/src-foundations/accessibility';
+
+export interface SelectProps
+	extends SelectHTMLAttributes<HTMLSelectElement>,
+		Props {
+	id?: string;
+	/**
+	 * Appears above the select box
+	 */
+	label: string;
+	/**
+	 * Adds the word "Optional" after the label
+	 */
+	optional?: boolean;
+	/**
+	 * Visually hides the label and the "Optional" text set by the `optional` flag.
+	 */
+	hideLabel?: boolean;
+	/**
+	 * Additional text that appears below the label
+	 */
+	supporting?: string;
+	/**
+	 * Whether error styling should apply to this select box. The string appears as an inline error message.
+	 */
+	error?: string;
+	/**
+	 * Whether success styling should apply to this select box. The string appears as an inline success message. This prop should not have a value set at the same time as the error prop. In the event that both are set, errors take precedence.
+	 */
+	success?: string;
+	children: JSX.Element | JSX.Element[];
```

Comment:
It's interesting that we sometimes go with `children: JSX.Element | JSX.Element[]` and sometimes go with `children: ReactNode`. I wonder if there's any useful distinction, or if we should adopt one consistently. Something for outside of this PR.

Reply:
Added a [Trello card](https://trello.com/c/BdgSmucL/416-standardise-children-types-for-react-components)

Reply:
Interestingly, if I comment out the children prop from the interface everything is still happy. It seems to be picking it up as `React.ReactNode` via the extension of `SelectHTMLAttributes<HTMLSelectElement>`. Maybe we should drop overriding it except for places where there's a particular need to do so?

Reply:
Yeah that’s a great idea!

Reply:
I'll update the card you've created as I suspect it's easy to do it as a wholesale change rather than trying to remember as we move the docs across.